tests/Predis/Connection/Aggregate/SentinelReplicationTest.php

```diff
@@ -1142,12 +1154,13 @@ public function testMethodExecuteCommandThrowsExceptionOnUnknownServiceName()
     }
 
     /**
-     * @group disconnected
-     * @expectedException Predis\ClientException
-     * @expectedExceptionMessage No sentinel server available for autodiscovery.
+     * @group disconnected.
```

Comment:
```suggestion
     * @group disconnected
```

Reply:
Good catch :eye:

Reply:
Fixed in ae551b9de12a723dd8caf1a7b60a7590530b1e42